src/ripple/net/impl/RPCCall.cpp

```diff
@@ -202,6 +218,25 @@ class RPCParser
         return v;
     }
 
+    Json::Value parseManifest (Json::Value const& jvParams)
+    {
+        if (1 == jvParams.size ())
```

Comment:
Style: We don't use yoda conditions in rippled. I understand the reasoning, but I'd like to stay consistent with the style of the rest of rippled.